vhdl/subsystems/vnir/LVDS_reader/LVDS_reader_IP.qip

```diff
@@ -0,0 +1,10 @@
+set_global_assignment -name IP_TOOL_NAME "ALTLVDS_RX"
```

Comment:
Is there an advantage to keeping these files around as opposed to just LVDS_reader_IP.vhd? If so they should probably be in their own folder or something

Reply:
Not sure that is one of the files generated automatically by Quartus.  I cleaned the project before uploading it so I assume any files that are still present are necessary.